samtranslator/validator/sam_schema/schema.json

```diff
@@ -371,6 +371,12 @@
         "Pattern": {
           "type": "object"
         },
+        "RuleName": {
+          "type": "string"
+        },
+        "State": {
+          "type": "string"
+        },
```

Comment:
Is this supposed to be here?

Reply:
leftover from other PR, I removed that commit from this branch/PR